src/Components/ImgBox.js

```diff
@@ -0,0 +1,181 @@
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+        <ImgColmnBox {...props} style={{ "margin-top": "250px" }}>
+          {ImgUrl &&
+            ImgUrl.map((item, index) => {
+              return (
+                index > 9 &&
+                index < 15 && (
+                  <ImgContent
+                    key={index}
+                    style={{ backgroundImage: `url(${item.url})` }}
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+        <ImgColmnBox {...props} style={{ "margin-top": "400px" }}>
+          {ImgUrl &&
+            ImgUrl.map((item, index) => {
+              return (
+                index > 14 &&
+                index < 20 && (
+                  <ImgContent
+                    key={index}
+                    style={{ backgroundImage: `url(${item.url})` }}
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+        <ImgColmnBox {...props} style={{ "margin-top": "250px" }}>
+          {ImgUrl &&
+            ImgUrl.map((item, index) => {
+              return (
+                index > 19 &&
+                index < 25 && (
+                  <ImgContent
+                    key={index}
+                    style={{ backgroundImage: `url(${item.url})` }}
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+        <ImgColmnBox {...props} style={{ "margin-top": "150px" }}>
+          {ImgUrl &&
+            ImgUrl.map((item, index) => {
+              return (
+                index > 24 &&
+                index < 30 && (
+                  <ImgContent
+                    key={index}
+                    style={{ backgroundImage: `url(${item.url})` }}
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+        <ImgColmnBox {...props}>
+          {ImgUrl &&
+            ImgUrl.map((item, index) => {
+              return (
+                index > 29 &&
+                index < 35 && (
+                  <ImgContent
+                    key={index}
+                    style={{ backgroundImage: `url(${item.url})` }}
+                  />
+                )
+              );
+            })}
+        </ImgColmnBox>
+      </ImgContainer>
+    </ImgDisplay>
+  );
+};
+
+export default ImgBox;
+
+////////////////////스타일드 컴포넌트///////////////////////////
+
+const ImgDisplay = styled.div`
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  position: absolute;
+  top: 0px;
+`;
+
+const ImgContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 200px 10px 0px 0px;
+  position: absolute;
+  left: 50%;
+  z-index: 5;
+  top: 0px;
+
+  width: 1750px;
+  height: 100vh;
+`;
+
+const ImgColmnBox = styled.div`
+  position: relative;
+  left: -50%;
+  width: 236px;
+  height: 250vh;
+  margin: 10px 10px 10px 10px;
```

Comment:
```suggestion
  margin: 10px;
```
헐! 당장 이렇게 줄여주세요!! ㅋㅋㅋㅋ

Reply:
> 헐! 당장 이렇게 줄여주세요!! ㅋㅋㅋㅋ

졸면서 코딩했나봐요. 수정했습니다

Reply:
@LontoJ 쉬엄쉬엄 하세요 ㅋㅋㅋㅋ